test/unit/modules/delegates.js

```diff
@@ -417,12 +417,12 @@ describe('delegates', () => {
 			});
 
 			it('should load secrets in encrypted format with the key', done => {
-				config.forging.secret = encryptedSecret;
+				config.forging.secret = encryptedPassphrase;
```

Comment:
`encryptedPassphrase` is being used in this file to describe sometimes an encrypted passphrase, and sometimes a list of items with an encrypted passphrase property. Can we make a clearer distinction?

Reply:
Follow up issue: https://github.com/LiskHQ/lisk/issues/1994